WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -145,6 +129,62 @@
           </div>
         <% end %>
         <div class="<%= collapse ? "collapse" : "" %>" id="registration_requirements_text">
+          <% if @competition.use_wca_registration? %>
+            <%= link_to t('.create_wca_account'), users_sign_up_path %>
+            <br>
+            <%= link_to t('.claim_wca_id'), profile_claim_wca_id_path %>
+            <br>
```

Comment:
nit: When possible, it's preferable to wrap things in `<p>...</p>` tags rather than scattering `<br>` tags around.

Reply:
Ok, I'll fix this.